<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArtistWindow</class>
 <widget class="QMainWindow" name="ArtistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Your Songs</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="2">
     <widget class="QLCDNumber" name="songNumber">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Add A Song</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="3">
     <widget class="QTableWidget" name="tableSongs">
      <column>
       <property name="text">
        <string>S/No</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Artist Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Song Name</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="ok">
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Remove Song</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Fixedsys</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Number of Owned Songs</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="4">
     <widget class="QListWidget" name="songsList"/>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>All Available Songs</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="addSong">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QLineEdit" name="removeSong"/>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="artist">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionLog_Out"/>
   </widget>
   <addaction name="menuUser"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log Out</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
